server/methods/getUsernameSuggestion.js

```diff
@@ -20,57 +17,37 @@ function usernameIsAvaliable(username) {
 	return !RocketChat.models.Users.findOneByUsername(username);
 }
 
+
+const name = (username) => (RocketChat.settings.get('UTF8_Names_Slugify') ? slug(username) : username);
+
 function generateSuggestion(user) {
 	let usernames = [];
-	let username = undefined;
 
-	if (Meteor.settings.public.sandstorm) {
-		usernames.push(user.services.sandstorm.preferredHandle);
-	}
+	// if (Meteor.settings.public.sandstorm) {
```

Comment:
if we'll officially not support sandstorm anymore, this can be removed and not commented out.

Reply:
by bad